local_detector:
    ros__parameters:
        inflation_window_size: 5